tests/utils/fileContents.ts

```diff
@@ -0,0 +1,18 @@
+import * as fs from "fs";
+import * as path from "path";
+
+export function replaceFileContents<T>(body: T, bundlePath: string): T {
```

Comment:
Let us not have this. If replacing file contents is the job of the runner, there is no point in adding that capability in tests. We can skip calling this function too.

Reply:
Makes sense, I'll do that.